### PR TITLE
Removed duplicates

### PR DESCRIPTION
**Pull request by Batre-MSFT**
Removed duplicate entries related to OWA and updated supportability stance w.r.t Outlook for Mac clients

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeHybrid/hybrid-deployment/set-up-modern-hybrid-public-folders.md [hunk 1]
```diff
@@ -46,18 +46,15 @@ An Exchange Online, Microsoft 365, or Office 365 user must be represented by a M
 
 2. These instructions assume that you have used the Hybrid Configuration wizard to configure and synchronize your on-premises and Exchange Online environments and that the DNS records used for most users' AutoDiscover references an on-premises end-point. For more information, see [Hybrid Configuration wizard](../hybrid-configuration-wizard.md).
 
-3. The public folders in this configuration cannot be accessed using Outlook Web App (Exchange 2013) or Outlook on the web (Exchange 2016 and Exchange 2019).
+3. The public folders in this configuration cannot be accessed using OWA (also referred as Outlook Web App or Outlook on the web).
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-deployment/set-up-modern-hybrid-public-folders.md:49]:
```suggestion
3. The public folders in this configuration cannot be accessed using OWA (also referred to as Outlook Web App or Outlook on the web).
```

File: Exchange/ExchangeHybrid/hybrid-deployment/set-up-modern-hybrid-public-folders.md [hunk 2]
```diff
@@ -46,18 +46,15 @@ An Exchange Online, Microsoft 365, or Office 365 user must be represented by a M
...
-
-   > [!NOTE]
-   > Outlook 2016 for Mac is supported for cross-premises public folders. If clients in your organization use Outlook 2016 for Mac, make sure they have installed the April 2016 update. Otherwise, those users will not be able to access public folders in a hybrid topology. For more information, see [Accessing public folders with Outlook 2016 for Mac](../../ExchangeOnline/collaboration-exo/public-folders/access-public-folders-with-outlook-2016-for-mac.md).
+7. Outlook 2016 for Mac (and higher versions) are supported for cross-premises public folders. If clients in your organization use Outlook 2016 for Mac, make sure they have installed the April 2016 or higher update installed. For more information, see [Accessing public folders with Outlook 2016 for Mac](../../ExchangeOnline/collaboration-exo/public-folders/access-public-folders-with-outlook-2016-for-mac.md).
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-deployment/set-up-modern-hybrid-public-folders.md:57]:
```suggestion
7. Outlook 2016 for Mac (and higher versions) are supported for cross-premises public folders. If clients in your organization use Outlook 2016 for Mac, make sure they have the April 2016 or higher update installed. For more information, see [Accessing public folders with Outlook 2016 for Mac](../../ExchangeOnline/collaboration-exo/public-folders/access-public-folders-with-outlook-2016-for-mac.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review. @Batre-MSFT, please commit my suggested changes if you agree with them. Many thanks.

**Comment by JohanFreelancer9** [comment]
@Batre-MSFT - Could you please commit my first suggested change as well (3. The public folders...)? Thanks.

**Pull request merged**